src/admin/components/Application/Application.tsx

```diff
@@ -0,0 +1,68 @@
+import Card from "react-bootstrap/Card";
+
+import { ApplicantActions, ApplicantStatus } from "admin/components";
+import {
+	Applicant,
+	ApplicationQuestion,
+	submitReview,
+} from "admin/utils/useApplicants";
+
+import ApplicationReviews from "./ApplicationReviews";
+import ApplicationSection from "./ApplicationSection/ApplicationSection";
+
+import styles from "./Application.module.scss";
+
+interface ApplicationSections {
+	[key: string]: ApplicationQuestion[];
+}
+
+const APPLICATION_SECTIONS: ApplicationSections = {
+	"Personal Information": ["gender", "pronouns", "ethnicity", "is_18_older"],
+	Education: ["university", "education_level", "major"],
+	Experience: ["portfolio_link", "linkedin_link", "resume_url"],
+	"Free Response Questions": [
+		"stress_relief_question",
+		"company_specialize_question",
+	],
+};
+
+interface ApplicationProps {
+	applicant: Applicant;
+	submitReview: submitReview;
+}
+
+function Application({ applicant, submitReview }: ApplicationProps) {
+	const { _id, application_data, status } = applicant;
+	const { first_name, last_name, submission_time, reviews } = application_data;
+
+	const submittedDate = new Date(submission_time).toDateString();
```

Comment:
Nitpick: I believe the reference point is still UTC time.

Reply:
Haven't noticed this myself, maybe differences in browser or OS compatibility? Running
```js
new Date("2023-01-12T00:06:00.000000+00:00").toDateString()
```

Should return `'Wed Jan 11 2023'`

Reply:
I think I may have missed that the conversion to local time is handled in the function call.